src/models/event.js

```diff
@@ -735,20 +735,34 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     },
 
     /**
-     * Get whether the event is a relation event, and of a given type if `relType` is passed in.
+     * Get whether the event is a relation event, and of a given type if
+     * `relType` is passed in.
      *
-     * @param {string?} relType if given, checks that the relation is of the given type
+     * @param {string?} relType if given, checks that the relation is of the
+     * given type
      * @return {boolean}
      */
     isRelation(relType = undefined) {
-        // must use event.content as m.relates_to is not encrypted
-        // and _clearEvent doesn't have it.
-        const content = this.event.content;
+        // Relation info is lifted out of the encrypted content when sent to
+        // encrypted rooms, so we have to check `getWireContent` for this.
+        const content = this.getWireContent();
         const relation = content && content["m.relates_to"];
         return relation && relation.rel_type && relation.event_id &&
             ((relType && relation.rel_type === relType) || !relType);
     },
 
+    /**
+     * Get relation info for the event, if any.
+     *
+     * @return {Object}
+     */
+    getRelation() {
```

Comment:
should we use this in `ReplyThread.js` as well?

Reply:
Unfortunately, we can't use it there at the moment because `isRelation` requires a `rel_type`, and current replies don't have a `rel_type`. I'll likely add a note about this in React SDK.